status.go

```diff
@@ -109,11 +110,12 @@ func StatusFormat() []byte {
 
 // StatusSet adds device to the status table or updates status
 // of the already known device
-func StatusSet(addr UsbAddr, desc UsbDeviceDesc, init error) {
+func StatusSet(addr UsbAddr, desc UsbDeviceDesc, init error, HTTPPort int) {
```

Comment:
Please, change the order of arguments, so error will be the last argument

Reply:
No problem.